core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -718,7 +692,7 @@ public List<Map<String, String>> selectActivityLogByTeam(
       for (Object[] actvty : activityCount) {
         hashMap = new HashMap<>();
         hashMap.put("dateofactivity", "" + actvty[0]);
-        hashMap.put("activitycount", "" + ((BigInteger) actvty[1]).intValue());
+        hashMap.put("activitycount", "" + ((Long) actvty[1]).intValue());
 
```

Comment:
is there an advantage to using Long v Integer if we are getting the intValue?

Reply:
Not an advantage, but jpa call is now returning Long instead of BigInteger, so had to change.